Mathlib/Data/Fintype/Defs.lean

```diff
@@ -139,6 +137,27 @@ def elabFinsetBuilderSetOf : TermElab
     elabTerm (← `(Finset.filter (fun $x:ident ↦ $p) (singleton $a)ᶜ)) expectedType?
   | _, _ => throwUnsupportedSyntax
 
+/-- Delaborator for `Finset.filter`. The `pp.piBinderTypes` option controls whether
```

Comment:
I'd say that this should be controlled by the funBinderTypes option, since that's the one which controls the same without this delaborator.